src/main/java/net/wurstclient/hacks/FullbrightHack.java

```diff
@@ -46,14 +49,23 @@ public FullbrightHack()
 		
 		EVENTS.add(UpdateListener.class, this);
 	}
-	
+
+	private static double gamma;
+
 	@Override
 	public void onUpdate()
 	{
+		if (!isEnabled()) {
+			if (MC.options.gamma <= 1 && smoothGamma == SmoothGamma.STOPPED) {
+				gamma = MC.options.gamma;

```

Comment:
...unless both of these conditions are true, the `gamma` field will stay at zero...

Reply:
Well, the idea is to update the saved gamma value when the player changes the gamma settings in game, but only if the smooth gamma ended and if value is lower or equals 1, being an extra check.